samples/JavaScript/entry.mjs

```diff
@@ -0,0 +1,6 @@
+import bar from './module.mjs';
```

Comment:
We prefer sample files from "real" applications under permissive license. These files are later used to train the Bayesian classifier so it's important that they be representative of actual use.